tests/chart/crds/runtime-rule-binding.crd.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I'm not sure about this, but we should consider on having the helm-chart repo as a submodule, this way we can have the CRD synchronized and with a single place of truth.

Reply:
In general, I would prefer to use the existing helm chart, however, it is not possible to deploy only the storage and the node-agent...(like it was possible in the past)...